app/styles/modules/_settings.scss

```diff
@@ -474,6 +474,16 @@ body.settings #main-content.card {
     width: 20%;
   }
 
+  .device-current {
+    color: $color-green;
+    font-size: 12px;
+    font-weight: bold;
+  }
+
+  .device-current + .last-connected {
```

Comment:
It seems like instead of using CSS to hide the last-connected span, we could just not add the element to the HTML:

```html
{{#isCurrentDevice}}
  <span class="device-current">{{#t}}current device{{/t}}</span>
{{/isCurrentDevice}}
{{^isCurrentDevice}}
  <div class="last-connected">
    {{lastAccessTime}}
 </div>
{{/isCurrentDevice}}
```